torchtune/models/qwen2_5/_model_builders.py

```diff
@@ -0,0 +1,1015 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import List, Optional
+
+from torchtune.data._prompt_templates import _get_prompt_template, _TemplateType
+
+from torchtune.models.qwen2._component_builders import lora_qwen2, qwen2
+from torchtune.models.qwen2_5._tokenizer import QWEN2_5_SPECIAL_TOKENS, Qwen2_5Tokenizer
+from torchtune.modules import TransformerDecoder
+from torchtune.modules.peft import LORA_ATTN_MODULES
+from torchtune.modules.tokenizers import parse_hf_tokenizer_json
+
+"""
+Model builders build specific instantiations using component builders. For example
+the qwen2_5_7b model builder uses the qwen2 component builder to create the
+Qwen2.5 7B model.
+"""
+
+
+def qwen2_5_0_5b() -> TransformerDecoder:
+    """
+    Builder for creating a Qwen2.5 model initialized w/ the default 0.5B parameter values
```

Comment:
is this both base and instruct?

Reply:
yes, for some reason some model sizes have different base/instruct architectures while others have the same. ill add a comment